data/gamerules.txt

```diff
@@ -88,3 +88,19 @@ gamerules
 	# DEFAULT: 1000 (1000 days)
 	# ALLOWABLE VALUES: any integer >= 0.
 	"depreciation max age" 1000
+	
+	# The minimum departure distance for all systems.
```

Comment:
For gamerules, it's better to err on the side of caution and thoroughly explain what the gamerule does, especially since the effect of arrival distance isn't immediately obvious and departure distance is only used for a handful of systems in the entire game.
```suggestion
	# The minimum departure distance for all systems. A ship can only jump out of a system if
	# it is further from the system center than the departure distance. If a system has a
	# defined departure distance greater than this value, that value will be used instead.
```